llvm/test/CodeGen/AIE/aie2/fdiv.ll

```diff
@@ -18,18 +18,18 @@ define bfloat @test_fdiv_bfloat(bfloat %a, bfloat %b) {
 ; CHECK-NEXT:    mova r0, #16; st lr, [sp, #-28] // 4-byte Folded Spill Delay Slot 3
 ; CHECK-NEXT:    st r16, [sp, #-32]; lshl r1, r1, r0 // 4-byte Folded Spill Delay Slot 2
 ; CHECK-NEXT:    lshl r2, r2, r0 // Delay Slot 1
-; CHECK-NEXT:    lda lr, [sp, #-28]; nopx // 4-byte Folded Reload
+; CHECK-NEXT:    lda lr, [sp, #-28] // 4-byte Folded Reload
+; CHECK-NEXT:    nop
```

Comment:
'ret' moved up, which is waiting for lr.

Reply:
Yes